File: burdine-brian/shortest-word.js
function findShort(s){
    console.log(s);
    let words = s.split(" ");
    let shortestLength = words[0].length;
    for (let word of words) {
      if (word.length < shortestLength) {
        shortestLength = word.length;
      }
    }
    console.log(shortestLength);
    return shortestLength;
  }
  
  /*
    Pseudocode
    START findShort
    READ string s
    SPLIT s into an array of words
    SET shortestLength = the length of the first word
    FOR every word in words
      IF the length of current word is less than the shortestLength
        SET shortestLength = length of current word
      ENDIF
    ENDFOR
    RETURN shortestLength
  */

File: burdine-brian/shortest-word.py
def find_short(str):
    words = str.split(" ")
    l = len(words[0])
    for word in words:
        if len(word) < l:
            l = len(word)
    return l 

#    print(str, ", Shortest word length: ", l)
# Pseudocode
# start find_short
# get s
# split s into an array of words
# init l as the length of the first
# for every word in words
#   if the length of word is less than l
#       set l to the length of word
#   end if
# endfor
# return l
# end find_short
# end find_short